.github/workflows/integration-tests.yml

```diff
@@ -20,15 +20,16 @@ jobs:
         working-directory: ./tests
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          distribution: 'zulu'
```

Comment:
Legacy version syntax 1.8 is now unsupported (it is equal to 8)
distribution parameter is now mandatory (zulu is set to get same behavior as v1)

Reference: https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md